### PR TITLE
Add request accessible format start page

### PR DESCRIPTION
**Pull request by edwardkerry**
This change adds a new Start Page to provide users with better
context and guidance on how to use the request accessible format
form.

It adds text to match the design and content provided.

Imported required styles for the inset-text component.

Moved the back-link component to the accessible formats requests layout,
this will render the back link underneath the header and appears consistent
with other rendering apps.

Co-Authored-By: Edward Kerry <edward.kerry@digital.cabinet-office.gov.uk>
Co-Authored-By: Martin Jones <martin.jones@digital.cabinet-office.gov.uk>

[trello](https://trello.com/c/x6QtEKU5/1145-introduce-a-start-page-to-request-accessible-format)

desktop-new

### CODE REVIEW

File: app/views/contact/govuk/accessible_format_requests/start_page.html.erb [hunk 1]
```diff
@@ -0,0 +1,50 @@
+<% content_for :title do %>
+  <% if flash[:input_errors]&.any? %>
```
Comment (KludgeKML) [inline app/views/contact/govuk/accessible_format_requests/start_page.html.erb:2]: Will flash work here, and... is it necessary? There aren't going to be any input errors on this page, are there? There's no input.

File: app/views/layouts/accessible_format_requests.html.erb [hunk 2]
```diff
@@ -10,6 +10,9 @@
   </head>
   <body>
     <div id="wrapper" class="govuk-width-container">
+    <% if @back_path %>
+      <%= render "govuk_publishing_components/components/back_link", { href: @back_path } %>
+    <% end %>
```
Comment (jon-kirwan) [inline app/views/layouts/accessible_format_requests.html.erb:15]: There may be an issue with moving the "Back" link to the layout view as it creates a larger gap between the link and the page title.

<table role="table">
<thead>
<tr>
<th>Before</th>
<th>After</th>
</tr>
</thead>
<tbody>
<tr>
<td valign="top"><a target="_blank" rel="noopener noreferrer" href="https://user-images.githubusercontent.com/87758239/164262093-afd7e6a2-759e-4a1a-8a19-c93847339169.png"></a></td>
<td valign="top"><a target="_blank" rel="noopener noreferrer" href="https://user-images.githubusercontent.com/87758239/164262110-a7959869-7cf3-4060-9a09-e490576c1722.png"></a></td>
</tr>
</tbody>
</table>

Also, the back link will now appear on the 'Request sent' page.

So, even though it's a bit repetitive, we should instead include the link in `start_page.html.erb` view.

**Review by jon-kirwan — CHANGES_REQUESTED** [review]

File: app/views/contact/govuk/accessible_format_requests/start_page.html.erb [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  text: sanitize(t("controllers.contact.govuk.accessible_format_requests.start_page.help_text"))
+} %>
+
+<%= form_with url: contact_govuk_request_accessible_format_format_type_path, method: :get, authenticity_token: false, html: { class: "contact-form" } do |f| %>
+  <%= hidden_field_tag :content_id, params["content_id"]%>
+  <%= hidden_field_tag :attachment_id, params["attachment_id"]%>
+
+  <%= render "govuk_publishing_components/components/button", {
+    text: "Start now",
+    start: true
```
[the remark this answers is not in the document]
Comment (MartinJJones) [reply app/views/contact/govuk/accessible_format_requests/start_page.html.erb:47]: Thanks, "Start now" has moved to the locale file

File: app/views/layouts/accessible_format_requests.html.erb (hunk 2 above)
Comment (MartinJJones) [reply app/views/layouts/accessible_format_requests.html.erb:15]: Thanks Jon, the back link is now included in each view where required.

File: app/views/contact/govuk/accessible_format_requests/start_page.html.erb (hunk 1 above)
Comment (MartinJJones) [reply app/views/contact/govuk/accessible_format_requests/start_page.html.erb]: Thanks Keith, it's not necessary and has been removed.

**Review by KludgeKML — APPROVED** [review]
LGTM

**Review by KludgeKML — APPROVED** [review]
Yep, seems good.

**Review by jon-kirwan — APPROVED** [review]
Looks good to me too 👍

**Pull request merged**